tests/test_fullscreen_plugin.py

```diff
@@ -5,8 +5,20 @@
 from porcupine import get_main_window
 from porcupine.menubar import get_menu
 
+github_actions = os.getenv("GITHUB_ACTIONS")
+xauthority = os.getenv("XAUTHORITY")
+
+headless = False
+
+if github_actions is not None:
+    headless = github_actions == "true"
+elif xauthority is not None:
+    headless = "xvfb" in xauthority
```

Comment:
Maybe:

```suggestion
headless = (
    os.getenv("GITHUB_ACTIONS") == "true" or "xvfb" in os.getenv("XAUTHORITY", "")
)
```

Reply:
...wow how did I not know about the second arg to `get`....